### DISCUSSION TITLE
Data for v0 tutorials

### DISCUSSION THREAD

**Discussion by sam9111** [Clay-foundation/model#234] [Q&A]
Hi. Super new and excited to try out Clay! I was just wondering what the data sources will be for the following Clay v0 tutorial notebooks:

1. CLAY v0 - Location Embeddings
2. CLAY v0 - Quality of reconstruction by the model
3. CLAY v0 - Interpolation between images

They all seem to have `DATA_DIR = "../data/02"` but how and where do I get this data from?

**Comment by tjhunter** [comment]
+1, it would be great to have a tiny example of a complete data generation pipeline for V1. I guess just adapting the [wall-to-wall example](https://clay-foundation.github.io/model/tutorials/clay-v1-wall-to-wall.html) to save the chips would be enough for the time being?

**Comment by srmsoumya** [comment]
Thanks for bringing this up @sam9111 & @tjhunter 

We have an example now to showcase data generation pipeline for Clay v1 - https://clay-foundation.github.io/stacchip/naip-tutorial/ thanks to @yellowcap!

@yellowcap , I think we should link this to Clay v1 docs or add a link.
